one-pager-maker/src/RouteAuthGuard.tsx

```diff
@@ -1,18 +1,36 @@
-import { Navigate, useLocation } from "react-router-dom";
+import {Navigate, useLocation} from "react-router-dom";
 import React from "react";
-import { useAppSelector } from "./redux/hooks";
+import {useAppSelector} from "./redux/hooks";
+import {Loader2} from "lucide-react";
```

Comment:
Why 2?

Reply:
Loader is dashed line circle

Reply:
I don't know. Why is it called `Loader2` for dashed line circle?

Reply:
This is it!
https://lucide.dev/icons/loader
https://lucide.dev/icons/loader-circle
In lucide-react library, Loader2 is icons/loader-circle and Loader is icons/loader